pwdtk/migrations/0005_auto_20200304_1116.py

```diff
@@ -46,6 +46,8 @@ class Migration(migrations.Migration):
             field=models.OneToOneField(
                 related_name='pwdtk_data',
                 null=True,
-                to=settings.AUTH_USER_MODEL),
+                to=settings.AUTH_USER_MODEL,
+                on_delete=models.CASCADE
```

Comment:
not necessary, but I would add a trailing `,` so that future changes will result only in a one-line-diff

Reply:
Done